### PR TITLE
Bead Marker (Xeros's first Abnormality)

### PR DESCRIPTION
**Pull request by X3ros**
An abnormality I made because I was bored. Sprite's shitty and code is probably of similar quality.

## About The Pull Request
This PR is mostly for the sake of receiving feedback and criticism regarding my first attempt at making an abnormality; I don't actually expect this to be merged. Also, if someone wishes, they can fix my awful abno sprite and spaghetti code if they actually want this thing in.

## Why It's Good For The Game
Adds another abnormality which might spice things up in the early game. Weapon is relatively unique as well.

## Changelog
:cl:
Added Bead Marker, a ZAYIN-Class abnormality with accompanying EGO gift, gear, and weapon.
EGO weapon is a hitscan pistol which (should) debuff red resistance.
Gear is simple RED 1 armor.
Gift adds 2 Justice.
Abnormality's gimmick is its ability to temporarily inflict Nearsightedness on agents which get a bad work result.
:cl:

### CODE REVIEW

File: code/modules/paperwork/records/info/zayin.dm [hunk 1]
```diff
@@ -127,3 +127,9 @@
 		"After work was completed, the fireflies glowed brighter.",
 		"The fireflies increased healing of HP and SP as long as the employee stayed in the room.",
 		"While the fireflies were glowing, no other work could be completed on O-02-183.")
+//Bead Marker
+/obj/item/paper/fluff/info/zayin/bead_marker
+	abno_type = /mob/living/simple_animal/hostile/abnormality/bead_marker
+	abno_code = "D-04-451"
```
Comment (Kitsunemitsu) [inline code/modules/paperwork/records/info/zayin.dm:133]: Do you happen to want to donate?

Comment (X3ros) [reply code/modules/paperwork/records/info/zayin.dm:133]: > Do you happen to want to donate?

I mean, I _could,_ but how does that relate to the PR?

**Comment by Coxswain-Navigator** [comment]
I suggest talking to other people to help make this a bit more interesting. We have forums on the discord for the express purpose of saving time by vetting these ideas before they are committed to code.

**Comment by X3ros** [comment]
> I suggest talking to other people to help make this a bit more interesting. We have forums on the discord for the express purpose of saving time by vetting these ideas before they are committed to code. If you're curious as to how it currently stands, it is unfortunately far below standard.

Understandable, I mostly did this for funsies and to see if I could actually make something of worth on my lonesome; again, I didn't really expect this to get merged. Thanks for the feedback, though.

**Comment by JackFrost2424** [comment]
1.) There's "cost =12" for the EGO gun, I would add a space there so it's "cost = 12" 
2.) Yeah there's nothing to actually say about the abno itself. It blinds you on a bad. That's it. Most ZAYINs need to have a pretty good gimmick to make people want to pick them and this feels like a barely formed concept. It doesn't even really feel like an abnormality design wise. A laser pointer...and that's it?

I dunno, I like the idea of blindness being a mechanic you have to deal with but that's all I can say.

File: code/modules/clothing/suits/ego_gear/zayin.dm [hunk 2]
```diff
@@ -108,5 +108,12 @@ Any attempt to code risk class armor will result in a 10 day Github ban.*/
 	icon_state = "oceanic"
 	armor = list(RED_DAMAGE = 10, WHITE_DAMAGE = 0, BLACK_DAMAGE = 0, PALE_DAMAGE = 0)
 
+/obj/item/clothing/suit/armor/ego_gear/zayin/pinpoint
+	name = "pinpoint"
+	desc = "An uncomfortable metal 'suit' with a glowing red dot in the center."
+	icon_state = "pinpoint"
+	armor = list(RED_DAMAGE = 10, WHITE_DAMAGE = 0, BLACK_DAMAGE = 0, PALE_DAMAGE = 0)
+
+
```
Comment (Gboster-0) [inline code/modules/clothing/suits/ego_gear/zayin.dm:119]: that is a lot of empty lines doing nothing, preferably the bottom of the file should only have a singular empty line

File: code/modules/mob/living/simple_animal/abnormality/zayin/bead_marker.dm [hunk 3]
```diff
@@ -0,0 +1,63 @@
+#define STATUS_EFFECT_BLINDPOINT /datum/status_effect/blindpoint
```
Comment (Gboster-0) [inline code/modules/mob/living/simple_animal/abnormality/zayin/bead_marker.dm:1]: preferably, put an empty line between
#define STATUS_EFFECT_BLINDPOINT /datum/status_effect/blindpoint
and
/mob/living/simple_animal/hostile/abnormality/bead_marker

File: code/modules/mob/living/simple_animal/abnormality/zayin/bead_marker.dm [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+/mob/living/simple_animal/hostile/abnormality/testabno/FailureEffect(mob/living/carbon/human/user, work_type, pe)
+	user.apply_status_effect(STATUS_EFFECT_BLINDPOINT)
+	to_chat(user, span_notice("Ack! That pointer shined a laser into your eyes!"))
+	..()
+
+
+
+	//This is my first time creating a custom abno, please pardon the spaghetti code and shitty spritework.
+
+
```
Comment (Gboster-0) [inline code/modules/mob/living/simple_animal/abnormality/zayin/bead_marker.dm:40]: again, jesus christ this is a lot of empty lines

**Review by Gboster-0 — COMMENTED** [review]
could use putting some trailing empty lines on the end of files, along with deleting some empty lines where they shouldnt be and putting some where they should

**Comment by X3ros** [comment]
To those who have commented on this, thank you for your feedback. This was, as stated before, not a serious attempt at an abnormality addition and more an experiment to see if I was capable of simply making something that would actually run. I'm currently working on an abnormality which has more of a defined gimmick and idea behind it. Expect me to poke my head in the discord asking for help with stuff I'm not familiar with soon.
This glorified test abnormality is being scrapped.

**Pull request closed without merging**